src/pages/TokenDetails/index.tsx

```diff
@@ -135,7 +135,7 @@ export default function TokenDetails() {
     <TokenDetailsLayout>
       {tokenAddress && (
         <>
-          <TokenDetail address={tokenAddress} />
+          <TokenDetail address={tokenAddress} connectedChainId={connectedChainId} />
```

Comment:
you can do `useWeb3React` in TokenDetail rather than needing to pass in connectedChainId here

Reply:
yeah but considering we are already getting it from the parent - 1 child down isn't that bad

Reply:
The page should not be concerned with the implementation details of TokenDetail component though.